install.sh

```diff
@@ -43,9 +43,9 @@ apt_install() {
     $sudo apt-get update || true
 
     # only install docker if podman can't be
-    if ! $sudo apt_get_install podman; then
+    if ! $sudo apt install podman; then
```

Comment:
`apt_get_install` is calling the function in line 37 to 39, so it should be changed there (and probably renamed).

Reply:
apt-get update should probably be changed to it's apt equivalent to be consistent also

Reply:
Also this should change:

available apt-get

to

available apt

Reply:
We do want -y 's in this script btw, the check to see if these things are installed in the "available" check